src/store/reducers/contracts.reducer.ts

```diff
@@ -30,7 +30,7 @@ export const contractsSlice = createSlice({
     },
 
     updateStatus: (state, action) => {
-      const idx = state.offers.findIndex((item) => item.id === action.payload.id);
+      const idx = state.offers.findIndex(item => item.id === action.payload.id);
```

Comment:
Suggestion: you can use `map()` and make a condition for one `id` and `action.payload.id` is the same to be what you want otherwise return the original element

Reply:
line 45 same
it's hard to read and maintain. I know you didn't change anything here but I appreciate you if you changed it.